timer.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
```

Comment:
@anonuevo-jheferson 
Suggestion:
Refer to the highlighted code.
You can just do this code instead to save code length:
`content="width=device-width, initial-scale=1.0"/>`

Reply:
> @anonuevo-jheferson Suggestion: Refer to the highlighted code. You can just do this code instead to save code length: `content="width=device-width, initial-scale=1.0"/>`

suggestion had been applied thank you!
https://github.com/PUP-BSIT/exercise-12-anonuevo-jheferson/blob/64ce7322d1d3fd99588eb758626268b7a69dc844/timer.html#L5

Reply:
This is acknowledged. Thank you!